app/src/main/java/app/revanced/manager/ui/screen/settings/AboutSettingsScreen.kt

```diff
@@ -48,14 +55,31 @@ fun AboutSettingsScreen(
     )
 
     val outlinedButton = listOf(
-        Triple(Icons.Outlined.Code, stringResource(R.string.github), third = {
+        Triple(FontAwesomeIcons.Brands.Github, stringResource(R.string.github), third = {
             context.openUrl("https://revanced.app/github")
         }),
         Triple(Icons.Outlined.MailOutline, stringResource(R.string.contact), third = {
             context.openUrl("mailto:nosupport@revanced.app")
         }),
     )
 
+    val socialIcons = mapOf(
+        "Contact" to Icons.Outlined.MailOutline,
+        "Discord" to FontAwesomeIcons.Brands.Discord, 
+        "Donate" to Icons.Outlined.FavoriteBorder,
+        "GitHub" to FontAwesomeIcons.Brands.Github,
```

Comment:
Since GitHub is hardcoded as an outlined button, this would show up twice. If we want to dynamically select the outlined/ filled buttons, we'd need to add them to the API response. Not sure how to solve this in another way

Reply:
This is what I mentioned before. Add an "important" value to social link

Reply:
We have something similar for the /donations endpoint that have `preferred` set to true for example.

Reply:
Yeah I was thinking something like that

Reply:
Waiting on https://github.com/ReVanced/revanced-api/pull/100

Reply:
Should be gtg.

Reply:
Should the "preferred" links use the outlined buttons or the filled ones.

Reply:
I believe, semantically, the filled ones are considered highlighted/preferred.